corehq/apps/case_importer_v1/util.py

```diff
@@ -139,10 +135,16 @@ def _row_values(self, sheet, index):
         return [self._fmt_value(cell) for cell in sheet.row(index)]
 
     def _fmt_value(self, cell):
-        # Explicitly format integers, since xlrd treats all numbers as decimals (adds ".0")
-        if cell.ctype == xlrd.XL_CELL_NUMBER and int(cell.value) == cell.value:
-            return int(cell.value)
-        return cell.value
+        if cell.ctype == xlrd.XL_CELL_NUMBER:
+            if int(cell.value) == cell.value:
```

Comment:
doesn't this also cast `1.0` to an int? also, do we do anything differently with `int`s and `floats` in cases?